<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Train</class>
 <widget class="QWidget" name="Train">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Train Model</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>581</width>
     <height>23</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>62</red>
         <green>180</green>
         <blue>137</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Link">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>62</red>
         <green>180</green>
         <blue>137</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Accent">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>62</red>
         <green>180</green>
         <blue>137</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>62</red>
         <green>180</green>
         <blue>137</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Link">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>62</red>
         <green>180</green>
         <blue>137</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Accent">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>62</red>
         <green>180</green>
         <blue>137</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Link">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>62</red>
         <green>180</green>
         <blue>137</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Accent">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>62</red>
         <green>180</green>
         <blue>137</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Train_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Train!</string>
   </property>
  </widget>
  <widget class="QComboBox" name="select_dataset">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Select Dataset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Settings">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QComboBox" name="select_model">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Select Model</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
